README.md

```diff
@@ -224,7 +224,7 @@ to encounter:
 * `-h`
 * `--help`
 * `--exact`
-* `--advanced`
+* `--advanced` <br>
```

Comment:
One should probably use a double line space here, or other markdown features to ensure a linebreak, rather than resorting to HTML